meetings/2021-03-15.md

```diff
@@ -0,0 +1,143 @@
+
+Dmitri: The confidential storage, then CCG call would be most useful. They are weekly.
+
+* https://github.com/decentralized-identity/confidential-storage/
+* https://lists.identity.foundation/g/sds-wg/wiki/18887/94176
+* https://lists.identity.foundation/g/sds-wg/wiki
+c
+Matthieu: See also https://w3c-ccg.github.io/meetings/
+
+Dmitri: it is good to join the CCG mailing list as the agendas for meetings appear there.
+https://www.w3.org/community/credentials/2020/09/
+
+confidential storage group (encrypted data vaults + identity hubs) - https://identity.foundation/confidential-storage/
+(that's the spec)
+github: https://github.com/decentralized-identity/confidential-storage/
+this one: https://lists.identity.foundation/g/sds-wg/wiki
+
+
+###  [Scope discussion](https://github.com/solid/authentication-panel/issues/86)
+
+Aaron: Requesting a specific scope of "solid" seems to be more in line with what OIDC does. It would require an extra burden for the client, but it doesn't seem too cumbersome.
+
+Elf: Would it be sufficient to have a webid claim to match what is in the access token's audience claim?
+
+Aaron: When you request the scope, then the ID token and the access token would include that claim and if you don't, it wouldn't be there. It's what happens with email claims for example.
+
+Dmitri: I think requiring a scope is a good idea, it sounds like a decent way to work with the mechanism.
+
+Aaron: Can we get feedback/concerns about client tooling for example?
+
+Henry: we could put a note in the spec linking to the issue so that people can be aware of the question we are asking.
+
+Matthieu: Spec level changelog? Would it be good to have those for all the Solid specs
+
+Aaron: I've seen it in other specs and it is useful.
+
+Sarven: It's specifically useful if you have significant snapshots of the spec (WD, CR, PR..)
+
+Henry: Could we use the git history?
+
+Aaron: There's a lot of info in history, it will be there among people fixing less significant things.
+
+
+### Our use of the webid claim in Solid-OIDC and how we can best support DIDs
+
+Aaron: How to best support DIDs? Right now we have a webid claim and we'd like to support dids. Even though the WebID says it must be an https URL, we sort of signal it could be a DID. We could be more specific and rename the claim to solid. So that would be future proof for supporting VCs, DIDs...
+
+Dmitri: +1 to Aaron's suggestion to have a general `solid` claim that specifies both WebID and DID.
+
+Matthieu: +1 to Aaron's suggestion.
+
+Justin: That makes a lot of sense.
+
+Aaron: I spoke with Ruben V. last week and he was very supportive, he suggested making the value space for this claim IRIs.
+
+Henry: Which DID do you support then? You could keep saying WebIDs because people argument they are more than HTTPS. That's a limitation we allowed to be formulated to finalize the spec (DID took another 7 years to mature).
```

Comment:
```suggestion
Henry: Which DID do you support then? You could keep saying WebIDs because people argument they are more than HTTPS. That's a limitation we allowed to be formulated to finalize the spec (DID took another 7 years to mature).
```
It should perhaps be noted that WebID spec has never been finalized; it remains a DRAFT document, and has never gone through full RecTrack process.

Reply:
It would be like saying that foaf:Person needs to go through full W3C standard track. WebID just is a URI with a particular referent following RDF rules.

Reply:
Actually I need to look at this write up of what I said, as it seems a bit mangled.